resin_release_tool/releaser.py

```diff
@@ -21,6 +23,71 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
+        all_apps = self.models.application.get_all()
+        if len(next((app for app in all_apps if app['id'] == int(self.app_id)),{})) != 0: #checks if the app exists
+            print("Found app with app_id ", self.app_id, "in Balena")
+            app_info = next((app for app in all_apps if app['id'] == int(self.app_id)),{})
+            all_devices = self.models.device.get_all_by_application_id(app_info['id'])
```

Comment:
you can get all the devices using, self.get_all_devices(), you don't need to search for the right app

Reply:
done!